server/methods/followUser.js

```diff
@@ -0,0 +1,43 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Users } from '../../app/models';
+import { settings } from '../../app/settings';
+
+
+Meteor.methods({
+	followUser(username) {
+		if (settings.get('Newsfeed_enabled')) {
+			// Update followers keys
+			let userObject = Users.findOneByUsername(username);
+			if ('followers' in userObject) {
+				const followersObject = userObject.followers;
+				const { _id } = Users.findOneByUsername(Meteor.user().username);
+				followersObject[_id] = '';
+				Users.update(userObject._id, { $set: { followers: followersObject } });
+			} else {
+				const followersObject = new Object();
```

Comment:
Why not validate at server startup and create this object there for every user to maintain consistency across all?